Source/UI/IdentityUI.swift

```diff
@@ -103,7 +103,7 @@ public class IdentityUI {
     // Used to store the currently presented identity process so that:
     // 1. The presentation of one process at a time can be enforced.
     // 2. When handling a universal link, the currently presented process (if any) can be retrieved.
-    private static var presentedIdentityUI: IdentityUI?
+    private weak static var presentedIdentityUI: IdentityUI?
```

Comment:
it would be better that in the commit mention *why* you need this to be weak, to understand the context, what are you trying to solve explicitly?

Reply:
True! I got lazy because this was a hot fix. The next release will have [this](https://github.com/schibsted/account-sdk-ios/pull/77) which does have a better commit message